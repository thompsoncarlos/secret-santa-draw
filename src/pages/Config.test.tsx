import { render } from '@testing-library/react'
import React from 'react'
import { RecoilRoot } from 'recoil'
import Config from './Config'

const mockNavigate = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavigate
    }
})

describe('The config page', () => {
    test('should be render correctly', () => {
        const { container } = render(
            <RecoilRoot>
                <Config />
            </RecoilRoot>
        )

        expect(container).toMatchSnapshot()
    })
})