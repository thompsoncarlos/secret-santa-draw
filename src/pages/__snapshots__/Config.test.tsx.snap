// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The config page should be render correctly 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Let's Start!
      </h2>
      <form>
        <div
          class="input-group-btn"
        >
          <input
            placeholder="Add participants name"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Add
          </button>
        </div>
        
      </form>
      <ul>
        ;
      </ul>
      <footer
        class="config-footer"
      >
        <button
          class="button"
          disabled=""
        >
          Start
        </button>
      </footer>
    </section>
  </div>
</div>
`;
